projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss

```diff
@@ -140,10 +145,21 @@
       box-shadow: none;
     }
   }
+
+  // From typography SECONDARY-13RG-exp == P5
+  @include mixins.css-var(font-size, clr-dropdown-font-size, inherit, variables.$clr-use-custom-properties);
+  @include mixins.css-var(letter-spacing, clr-dropdown-letter-spacing, inherit, variables.$clr-use-custom-properties);
+  @include mixins.css-var(line-height, clr-dropdown-line-height, inherit, variables.$clr-use-custom-properties);
 }
 
 @mixin generate-dropdown-item-height() {
   padding: variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_1;
```

Comment:
We probably don't need that with the mixin fallback below/

Reply:
Yes, thanks! Forgot to remove it.